auto_nag/scripts/duplicate_copy_metadata.py

```diff
@@ -0,0 +1,172 @@
+                        }
+                    elif new_access_tag == copied_fields["whiteboard"]["value"]:
+                        copied_fields["whiteboard"]["from"].append(dup_bug["id"])
+
+            if copied_fields:
+                copied_fields = sorted(
+                    (
+                        field,
+                        change["value"],
+                        utils.english_list(
+                            sorted(f"bug {bug_id}" for bug_id in change["from"])
+                        ),
+                    )
+                    for field, change in copied_fields.items()
+                )
+
+                results[bug_id] = {
+                    "id": bug_id,
+                    "summary": bug["summary"],
+                    "copied_fields": copied_fields,
+                }
+
+                self.set_autofix(bug, copied_fields)
+
+        return results
+
+    def set_autofix(self, bug: dict, copied_fields: List[tuple]) -> None:
+        """Set the autofix for a bug
+
+        Args:
+            bug: The bug to set the autofix for.
+            copied_fields: The list of copied fields with their values and the
+                bugs they were copied from (field, value, source).
+        """
+        bug_id = str(bug["id"])
+        autofix = {}
+
+        duplicates = {source for _, _, source in copied_fields}
+        comment = (
+            f"The following {utils.plural('field has', copied_fields, 'fields have')} been copied "
+            f"from {utils.plural('a duplicate bug', duplicates, 'duplicate bugs')}:\n"
+            "| Field | Value | Source |\n"
+            "| ----- | ----- | ------ |\n"
+        )
+
+        for field, value, source in copied_fields:
+            if field == "keywords":
+                autofix["keywords"] = {"add": value}
+            elif field == "whiteboard":
+                autofix["whiteboard"] = bug["whiteboard"] + value
+            else:
+                raise ValueError(f"Unsupported field: {field}")
+
+            comment += f"| {field.capitalize()} | {value} | {source} |\n"
+
+        comment += "\n\n" + self.get_documentation()
+        autofix["comment"] = {"body": comment}
+        self.autofix_changes[bug_id] = autofix
+
+    def columns(self):
+        return ["id", "summary", "copied_fields"]
+
+    def get_bz_params(self, date):
+        fields = [
+            "whiteboard",
+            "keywords",
+            "dupe_of",
+        ]
+
+        params = {
+            "include_fields": fields,
+            "resolution": "DUPLICATE",
+            "chfieldfrom": "-7d",
+            "chfield": [
+                "resolution",
+                "keywords",
+                "status_whiteboard",
+            ],
+            "j1": "OR",
+            "f1": "OP",
+            "f3": "status_whiteboard",
+            "o3": "anywordssubstr",
+            "v3": "[access-s",
+            "f4": "keywords",
+            "o4": "equals",
+            "v4": "access",
```

Comment:
might not need this if we just rely on the whiteboard

Reply:
It is part of an `OR` condition. The goal here to copy the `access` keyword if it it missed, even if the bug does not have `[access-s*]` on the whiteboard.